pkg/maps/lbmap/lbmap.go

```diff
@@ -370,7 +370,13 @@ func updateMasterService(fe ServiceKey, nbackends int, revNATID int, svcType loa
 	zeroValue := fe.NewValue().(ServiceValue)
 	zeroValue.SetCount(nbackends)
 	zeroValue.SetRevNat(revNATID)
-	zeroValue.SetFlags(loadbalancer.CreateSvcFlag(svcLocal, sessionAffinity, !fe.IsSurrogate(), svcType).UInt8())
+	flag := loadbalancer.NewSvcFlag(&loadbalancer.SvcFlagParam{
```

Comment:
unrelated to this PR, I feel like this method can be changed to take struct argument as well https://github.com/cilium/cilium/blob/4f73469c9e32f5ca0e47670fe443272366cc5ad5/pkg/maps/lbmap/lbmap.go#L49

Reply:
Yeah, it's coming. I didn't want to touch the lbmap pkg in this PR.

Reply:
nice :100:, sorry to jump in too soon